pipeline/transformer/transformer_mets/src/test/resources/mappings/mapping-oauth2-token.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "c4c51257-4385-4543-8559-50d46241ee46",
+  "request": {
+    "url": "/oauth2/token",
+    "method": "POST",
+    "headers": {
+      "Authorization": {
+        "equalTo": "Basic Y2xpZW50OnNlY3JldA=="
```

Comment:
We don't care about this being there?

Reply:
it's `client:secret`. Not the real credentials